Fix bug alien vault otx url error response (#12090)
demisto/content: Packs/AlienVault_OTX/ReleaseNotes/1_0_4.md

#### Integrations
##### AlienVault OTX v2

>  - Fixed an issue in which the ***url*** command for urls with status code 404 return no matches instead of error 

Reviewer: ```suggestion
 - Fixed an issue where the ***url*** command failed with status code 404.
```
